README.md

```diff
@@ -46,6 +48,8 @@ tcli.exe run {game identifier}
 ```
 The `--profile` snippet from above still applies to both of those commands.
 
+If you want to run the game with a specific set of arguments, you can use `--args "--flag parameter1 parameter2"`
```

Comment:
Isn't it relatively common convention to have `--` break the arg parsing & consider everything after as something that should be passed to the target directly? For example, `tcli.exe run valheim -- --foo bar`

Reply:
I could probably do so by adding some handling for it before passing it into the command line parser, just didn't think about it at the time. I can swap it out in a second PR if you think it's more appropriate

Reply:
`--` is my preference because of the string nesting issue @MythicManiac described. `--args` is more explicit but it's nothing that can't be fixed with an explanation in the help command.